compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -55,6 +48,23 @@ static llvm::cl::opt<std::string> clPathToUkernels(
     "iree-amdaie-path-to-ukernels",
     llvm::cl::desc("Path to microkernels' directory"));
 
+static llvm::cl::opt<bool> clEnableVectorizationInPassPipeline(
+    "iree-amdaie-enable-vectorization-in-pass-pipeline",
```

Comment:
I think flag name `iree-amdaie-enable-vectorization` is enough?

Reply:
Ok. I actually initially had that, but then thought it was weird that something like 

```
iree-opt --iree-amdaie-enable-vectorization=0 --iree-amdaie-vectorization
```

does vectorize. But on seconds thoughts it's a weird thing to do. I'll change it tomorrow

Reply:
Okay, I see. Then maybe `iree-amdaie-enable-vectorization-passes`?

Reply:
A good compromise, I'll run with that.